kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js

```diff
@@ -267,7 +250,12 @@ define([
                         },
                         [
                             // indicates the job status when the cell is collapsed
-                            collapsedCellJobStatus,
+                            span(
+                                {
+                                    dataElement: 'job-status',
+                                },
+                                collapsedCellJobStatus
+                            ),
```

Comment:
minor structural change -- the 'job-status' element is always there, but it doesn't have content unless the cell is minimised